### ISSUE TITLE
Saved corrupt tour

### ISSUE THREAD

**Issue by doctorspaceman** [WorldWideTelescope/wwt-web-client#146]
I made a tour with music on a master first slide and two slides with audio per-slide voiceover.  I made the tour with Windows/Chrome.  I tried to read it back into Chrome and it failed.  Tried Windows/Edge and the Desktop client all failed.

[New WebGL Tour!.zip](https://github.com/WorldWideTelescope/wwt-web-client/files/470804/New.WebGL.Tour.zip)

**Comment by thewebkid** [comment]
This tour won't open in the windows client. The first node has an empty overlays element that is causing the issue. I am assuming this was created in the web client. I added a catch in the Tourstop.fromXml that will show the invalid tour popup. We might want to keep an eye on this issue if we find any other tours created in the web client with empty overlays elements that cause errors.
